Kernel/Arch/CurrentTime.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2022, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Types.h>
+
+namespace Kernel {
+
+typedef u64 (*fptr)();
```

Comment:
Can this be done more idiomatically, i.e
 via Function and using?

Reply:
Unfortunately no, I tried that and the compiler insists that __dso_handle is not implemented when I try to change things to work with the `Function` container, due to a required change in the `Scheduler.cpp` file:
https://github.com/SerenityOS/serenity/blob/0c07bed89c58d3fca745fbe12753335247406c05/Kernel/Scheduler.cpp#L53-L55

When I change that line to be `Function<u64(void)> Scheduler::current_time;` it doesn't work with the __dso_handle error.

Reply:
Turns out it is too much hacky and ugly, so I will leave it this way.